src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private final Colors colorInColors = Colors.WHITE;
+    private String color = colorInColors.name();
+
```

Comment:
with your current implementation any figure only able to be with `Colors=Colors.WHITE`. 

use constructor for initializing object.